lib/helpers.js

```diff
@@ -24,4 +28,56 @@ async function getDirectories (rootPath) {
   return dirs.sort();
 }
 
-export { getDirectories };
+async function getAndroidPlatformAndPath () {
+  const androidHome = process.env.ANDROID_HOME;
+  if (typeof androidHome !== "string") {
+    log.error("ANDROID_HOME was not exported!");
+    return null;
+  }
+  // TODO import it from appium support
+  let locs = ['android-4.2', 'android-17', 'android-4.3', 'android-18',
+      'android-4.4', 'android-19', 'android-L', 'android-20', 'android-5.0',
+      'android-21'];
+
+  for(let loc of locs.reverse()) {
+    let platforms = path.resolve(androidHome, 'platforms');
+    if(await util.fileExists(path.resolve(platforms, loc))) {
+      return {platform: loc, platformPath: path.resolve(platforms, loc)};
+    }
+  }
+  return null;
+}
+
+async function unzipFile (zipPath) {
+  log.debug(`Unzipping ${zipPath}`);
+  try {
+    await assertZipArchive(zipPath);
+    if (system.isWindows()) {
+      let zip = new AdmZip(zipPath);
+      zip.extractAllTo(path.dirname(zipPath), true);
+      log.debug("Unzip successful");
+    } else {
+      let execOpts = {cwd: path.dirname(zipPath)};
+      await exec('unzip', ['-o', zipPath], execOpts);
+      log.debug("Unzip successful");
+    }
+  } catch (e) {
+    throw new Error(`Error occored while unzipping. Original error ${e.message}`);
```

Comment:
occored --> occurred

and I would say `Original error: ${e.message}` so it reads more cleanly (i.e., add a colon)